db/upgrade.php

```diff
@@ -182,6 +115,73 @@ function xmldb_tincanlaunch_upgrade($oldversion) {
         upgrade_mod_savepoint(true, 2015033100, 'tincanlaunch');
     }
 
+    if ($oldversion < 2015112702) { //New version in version.php
```

Comment:
I did not. just copy pasted your code and re order it. (indeed weird)

install.xml is only used when anyone first installs the module. and as I was upgrading, Moodle did not even read it.
of course, you should keep it updated at all times, for the sake of anyone who installs the plugin for the first time.